source/backend/api/Services/AcquisitionFileService.cs

```diff
@@ -227,14 +227,19 @@ public PimsAcquisitionFile Add(PimsAcquisitionFile acquisitionFile, IEnumerable<
             ValidateStaff(acquisitionFile);
             ValidateOrganizationStaff(acquisitionFile);
 
-            acquisitionFile.AcquisitionFileStatusTypeCode = "ACTIVE";
+            if (acquisitionFile.PimsPropertyAcquisitionFiles.Any(x => x.Property.IsRetired.HasValue && x.Property.IsRetired.Value))
```

Comment:
hmm, yes. This appears to only affect add but I imagine it should affect update as well?

Reply:
Unless you were expecting the repository to handle file update adding new properties? in that case the lease property probably needs something similar.

Reply:
The adding does not call the AcquisitionFilePropertyRepository methods, so the check had to be added in the Service. 
For the update I am taking into consideration the developer note that states: "Although if a property is already on a file, it can be ignored during this process." So it's only when "UpdatingProperties" and the flow is to "Add".